pom.xml

```diff
@@ -310,6 +316,26 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
```

Comment:
Does this dependency scope should be runtime?

Reply:
Thanks for pointing this